content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -19,10 +19,10 @@ For information how to create a cluster with kubeadm once you have performed thi
 
 * One or more machines running one of:
   - Ubuntu 16.04+
-  - Debian 9
-  - CentOS 7
-  - RHEL 7
-  - Fedora 25/26 (best-effort)
+  - Debian 9+
+  - CentOS 7+
+  - Red Hat Enterprise Linux (RHEL) 7+
```

Comment:
I don't believe RHEL 8 supports iptables-legacy at all. See https://access.redhat.com/solutions/4377321 which states that legacy support was dropped. "No, it doesn't. iptables source rpm includes iptables/xtables-legacy-multi.c, however the rpmspec drops legacy ones." if you can't access doc. SO we should probably state only for RHEL 7 and, er, say that Kubernetes is not supported on RHEL 8?